app/src/components/menu.ts

```diff
@@ -305,55 +339,58 @@ export function createMenu({
     menuTemplate = [editMenu, viewMenu, windowMenu, helpMenu];
   }
 
+  return menuTemplate;
+}
+
+function injectBookmarks(menuTemplate: MenuItemConstructorOptions[]): void {
+  const bookmarkConfigPath = path.join(__dirname, '..', 'bookmarks.json');
+
+  if (!fs.existsSync(bookmarkConfigPath)) {
+    return;
+  }
+
   try {
-    const bookmarkConfigPath = path.join(__dirname, '..', 'bookmarks.json');
-    if (fs.existsSync(bookmarkConfigPath)) {
-      const bookmarksMenuConfig: BookmarksMenuConfig = JSON.parse(
-        fs.readFileSync(bookmarkConfigPath, 'utf-8'),
-      );
-      const bookmarksMenu: MenuItemConstructorOptions = {
-        label: bookmarksMenuConfig.menuLabel,
-        submenu: bookmarksMenuConfig.bookmarks.map((bookmark) => {
-          if (bookmark.type === 'link') {
+    const bookmarksMenuConfig: BookmarksMenuConfig = JSON.parse(
+      fs.readFileSync(bookmarkConfigPath, 'utf-8'),
+    );
+    const bookmarksMenu: MenuItemConstructorOptions = {
+      label: bookmarksMenuConfig.menuLabel,
+      submenu: bookmarksMenuConfig.bookmarks.map((bookmark) => {
+        switch (bookmark.type) {
+          case 'link':
             if (!('title' in bookmark && 'url' in bookmark)) {
-              throw Error(
+              throw new Error(
```

Comment:
Surprise ESLint didn't chime about this. Can we enable a rule to make it error?

Reply:
Unfortunately I don't see a rule that covers this:

https://eslint.org/docs/rules/